### PR TITLE
store: clean up event.go, node.go and add tests

### PR DESCRIPTION
**Pull request by gyuho**
Updates IsCreated logic on event.go. Cleans up node.go
and adds tests to it.

### CODE REVIEW

File: store/node_test.go [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+package store
+
+import (
+	"testing"
+	"time"
+
+	"github.com/coreos/etcd/Godeps/_workspace/src/github.com/jonboulle/clockwork"
+)
+
+func TestNewKVOperations(t *testing.T) {
```
Comment (xiang90) [inline store/node_test.go]: I feel these two tests are too long. I would suggest to separate them into smaller tests that only test one functionality a time.

**Comment by xiang90** [comment]
The changes look good to me.

File: store/node_test.go (hunk 1 above)
Comment (gyuho) [inline store/node_test.go]: Just split them. Thanks

**Comment by gyuho** [comment]
@xiang90 Will merge later today, after test pass, if you don't have objection.
But going forward, I will focus on `storage` package. Thanks for review.

**Comment by lycon2014** [comment]
Test milipeade as a binary infusion/injunction or stack node

**Pull request merged**